### ISSUE TITLE
esprima-harmony@7001.1.0-dev-harmony-fb deprecated

### ISSUE THREAD

**Issue by k0954342** [js2coffee/js2coffee#343]
I have cloned the repo and did a ```sudo npm install``` inside my ```usr/lib/node_modules/js2ccoffee``` directory. When doing this I received 

```
npm WARN deprecated esprima-harmony@7001.1.0-dev-harmony-fb: use esprima-fb instead
```
Should the dependencies need to be updated?

**Comment by rstacruz** [comment]
good catch... that's a recent deprecation on their part AFAIK.

**Comment by rstacruz** [comment]
btw, there's on need to `sudo` in `npm install` :) I recommend doing `npm install && npm link` if you want the `js2coffee` bin globally available from your dev instance.

**Comment by k0954342** [comment]
@rstacruz thanks. i'll close the issue.

**Comment by rstacruz** [comment]
nah lets keep it open until js2coffee migrates out of `esprima-harmony`. PR's welcome until I can attend to this around the weekend.

**Comment by rstacruz** [comment]
landed in v2.0.1, enjoy.
